### DISCUSSION TITLE
關於演算法

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#596] [Show and tell]
https://www.techbang.com/posts/22611-algorithm-of-everyday-life-the-ubiquitous-stealth-force#:~:text=%E8%80%8C%E5%9C%A8%E4%BA%A4%E6%98%93%E9%87%8F%E9%BE%90%E5%A4%A7,%E7%9A%84%E4%BA%A4%E6%98%93%EF%BC%8C%E6%95%99%E4%BA%BA%E5%98%86%E7%82%BA%E8%A7%80%E6%AD%A2%E3%80%82
相信很多人在看社群的時候，常常因為演算法接觸到許多對於自己有興趣的東西，且也能有對此更多的了解，我也深受其中的恩惠，YOUTUBE的影片看到類似的都能看個許久，在查找資料時也特別方便，但是每當自己在看文章的時候都會想想，我們是不是也在漸漸地被演算法操控了思想，我們對於想像也會開始有了限制，久而久之也是一件蠻嚴重的事，但不能否忍演算法帶給我們許多的便利，很難想像往後又沒有了演算法的網路世界。

**Comment by HsuChen88** [comment]
身為人還是得壓抑自己的右手不要點進去
不然就跟機器一樣被控制了

**Comment by hapiwangy** [comment]
現在的演算法的確是越來越神奇了，也正在影響著我們生活的趨向

**Comment by handsomeniko** [comment]
演算法真的控制我們的生活

**Comment by Sazmafuji** [comment]
演算法主宰我們世界萬物，matrix

**Comment by yucheng1028** [comment]
我現在已經對演算法感到害怕了，他怎都知道我想買甚麼，絕對不是我腦波弱

**Comment by jason79461385** [comment]
每次跟朋友聊完某個東西後，就自動會跳出那個東西的廣告，真的蠻可怕的呢
